Sources/ReleaseServer/ReleaseServer.WebApi/ReleaseServer.WebApi.csproj

```diff
@@ -2,13 +2,17 @@
 
     <PropertyGroup>
         <TargetFramework>netcoreapp3.1</TargetFramework>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <NoWarn>$(NoWarn);1591</NoWarn>
```

Comment:
We prefer to not suppress any kind of warnings if not absolutely necessary. Why do want to suppress this one?

Reply:
This warning generates >50 warnings for classes / functions without a XML documentation.
So that i don't need to document every single function, i deactivated the warning.

How should i proceed further with this warning?

Reply:
This happens due to setting `<GenerateDocumentationFile>true</GenerateDocumentationFile>`, doesn't it? Is this setting needed for using Swashbuckle?

Reply:
Yes.
It's needed generally. The Swagger frameworks (Swashbuckle & NSwag) need it, to render the descriptions into the Swagger documentation.

Reply:
Without a documentation file -> no descriptions in Swagger

Reply:
As @f-porter already pointed out: Warnings shall be solved and not ignored.
Solution: Change the access modifiers of the API to internal, which is not to be accessed from others.

Reply:
Thanks for the hint.

I will refactor the access modifiers and complete the XML documentation for the classes / functions in the course of #34. This was planned as next step anyway :)
 
So, the suppression is a temporary situation.